cypress/support/selectors.js

```diff
@@ -0,0 +1,28 @@
+class Selectors {
```

Comment:
Same here, better to split Selectors and commands across pages. Like in the Page Object Model.